Library/Homebrew/cmd/update-report.rb

```diff
@@ -250,10 +250,23 @@ def migrate_tap_migration
       tabs = dir.subdirs.map { |d| Tab.for_keg(Keg.new(d)) }
       next unless tabs.first.tap == tap # skip if installed formula is not from this tap.
       new_tap = Tap.fetch(new_tap_name)
-      new_tap.install unless new_tap.installed?
-      # update tap for each Tab
-      tabs.each { |tab| tab.tap = new_tap }
-      tabs.each(&:write)
+      # related to formulae deduplication. If formula stays in cask, autoinstall or provide cask instructions
+      if new_tap_name == "caskroom/cask"
+        if new_tap.installed? && File.exist?("/usr/local/Caskroom")
```

Comment:
I'm aware of this, but still: [According to this piece of code](https://github.com/vitorgalvao/homebrew-cask/blob/377c29cfa331c5ce7ea52e818a0c57e7017ff5a4/lib/hbc/locations.rb#L11-L13) the new default location is not hard-coded to use `/usr/local` but wherever Homebrew happens to be installed (and this doesn't have to be `/usr/local`).